core/vm/vm.go

```diff
@@ -201,6 +199,35 @@ func (self *Vm) Run(contract *Contract, input []byte) (ret []byte, err error) {
 
 						continue
 					}
+				case CREATE:
```

Comment:
Why did this get added to the switch statement? Is it because of `params.IsHomestead`?